safe/gui/tools/test/test_field_mapping_dialog.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+
+"""Tests for field mapping dialog."""
+
+import unittest
+import logging
+
+from safe.test.utilities import get_qgis_app, load_test_vector_layer
+from safe.common.exceptions import KeywordNotFoundError
+
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+
+from safe.definitions.layer_purposes import layer_purpose_aggregation
+from safe.definitions.fields import female_ratio_field
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->